### ISSUE TITLE
Betas normalization

### ISSUE THREAD

**Issue by smandava98** [statho/ScoreHMR#5]
Hey! Great work!

I just had one question:

Where did you get the min and max values to normalize the betas from? I see the file name is betas_stats_eft_fits.npz. Where is this initially derived from? Did you calculate this or?

**Comment by statho** [comment]
The `min` and `max` values for the SMPL betas were calculated from the pseudo ground-truth betas of the training datasets. The idea behind this normalization is to put the SMPL betas in [-1, 1] range, so that we can use common variance schedules for the diffusion process. The SMPL thetas are naturally within that range.

We found that using unnormalized SMPL betas works well too!
